### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by feriljohn**
changed the component names

### CODE REVIEW

File: diagramming-tool/.gitignore
Comment (AnushiyaM11) [inline diagramming-tool/.gitignore:1]: Can you remove the Head and main

File: diagramming-tool/package.json [hunk 1]
```diff
@@ -3,32 +3,22 @@
...
-
-    "@testing-library/user-event": "^13.5.0",
-    "@types/react": "^18.2.66",
-
-    "mdb-react-ui-kit": "^7.2.0",
     "react": "^18.2.0",
+    "react-dnd": "^16.0.1",
+    "react-dnd-html5-backend": "^16.0.1",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.22.3",
```
Comment (AnushiyaM11) [inline diagramming-tool/package.json:19]: react-router-dom is needed for redirection

File: diagramming-tool/src/App.jsx [hunk 2]
```diff
@@ -1,30 +1,13 @@
...
 function App() {
   return (
-
-    <Router>
-      <div className="App">
-        <header className="App-header">
-         
-        </header>
-        <Routes>
-          <Route exact path="/" element={<Home/>} />
```
Comment (AnushiyaM11) [inline diagramming-tool/src/App.jsx:20]: Routes should not be removed

File: diagramming-tool/src/setupTests.js [hunk 3]
```diff
@@ -1,26 +1,2 @@
...
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
-
-// setupTests.js
 import '@testing-library/jest-dom';
-class IntersectionObserver {
-    constructor() {}
-  
-    observe() {
```
Comment (AnushiyaM11) [inline diagramming-tool/src/setupTests.js:11]: I think these set of code is needed for login testing please do checl

**Review by AnushiyaM11 — APPROVED** [review]

**Review by Sharz-dev — APPROVED** [review]

**Review by AnushiyaM11 — APPROVED** [review]

**Pull request merged**